cmd/osv-scanner/fix/main.go

```diff
@@ -29,12 +29,12 @@ const (
 )
 
 type osvFixOptions struct {
-	remediation.RemediationOptions
+	remediation.Options
 	Client     client.ResolutionClient
 	Manifest   string
-	ManifestRW manifest.ManifestIO
+	ManifestRW manifest.IO
 	Lockfile   string
-	LockfileRW lockfile.LockfileIO
+	LockfileRW lockfile.IO
```

Comment:
I've had the tendency to name these variables `rw` (for read-write).
It might make sense to rename the structs `manifest.ReadWriter` and `lockfile.ReadWriter`.
Any thougts/opinions?

Reply:
I don't have preference between the two options, both SGTM.

Reply:
I've gone and renamed them to `ReadWriter`s